### DISCUSSION TITLE
Auto update button for all Python modules

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7820] [Enchancements]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Auto-update-button-for-all-Python-module/10560815)._

---
![Python modules autoupdate button request.png](https://aka.ms/dc/image?name=B1f4ff51c34bd4ee3872cbe9fafb4586c638407533054346726_Python_modules_autoupdate_button_request.png&tid=1f4ff51c34bd4ee3872cbe9fafb4586c638407533054346726)

Can a button to autoupdate all Python modules for a given version in Python Environments be added in the UI? It will save a tremendous amount of tedious manual upkeeping repeatedly! 

---
#### Feedback Bot on 1/14/2024, 11:52 PM: 

(private comment, text removed)

**Comment by StellaHuang95** [comment]
Sounds like a reasonable ask. Considering moving to discussion. What's your thoughts? @cwebster-99
